ecosystem/indexer-grpc/indexer-transaction-generator/README.md

```diff
@@ -18,4 +19,21 @@ import_config:
     # A map from versions to dump and their output names.
     versions_to_import:
       123: testnet_v1.json
+# Config to generate the transactions via localnode.
+script_transaction_generator_config:
```

Comment:
What is the motivation for having a centralized config for the scripted transactions? Centralized config makes sense for imported transactions, but it's difficult to manage when there's multiple PRs touching this file and results in merge conflicts. 

Another option is to have a central folder where all the scripts live. The transaction generator can pickup code changes in that folder and regen the transactions if needed.

Reply:
the other option doesn't work since there may be dependent transactions and if we put all dependent transactions under the same folder, we lose the sharable steps.

Reply:
The conflicts are expected for both importing and scripting; this is managable if we append these runs into the config.

Reply:
One solution can be:
1. move_folder to hold all scripts
2. multiple config files for script transactions. like
```
# fund, transfer and burn.
transactions:
   - name: fund
      path:
   - name: transfer
      path:
   - name: burn
      path: 
```
```
# Create NFT and transfer
transactions:
  - name: create
     path: 
  - name: transfer
     path: 
```

Reply:
if we name the move script properly, this seems easier to manage to me.

Reply:
qq: the shared move scripts might be using different account addresses, how does it know which address to use in the script? will the addresses be set in config file?

Reply:
no the addresses are hardcoded in the config/move files. 

To allow higher flexibility, we need to add support for addresses to be configured and or flexible script args with which we can pass during runtime. This can be a follow-up task.

Reply:
done.